airflow/providers/microsoft/azure/sensors/wasb.py

```diff
@@ -26,6 +26,13 @@
 from airflow.providers.microsoft.azure.triggers.wasb import WasbBlobSensorTrigger, WasbPrefixSensorTrigger
 from airflow.sensors.base import BaseSensorOperator
 
+try:
+    from airflow.models.abstractoperator import DEFAULT_DEFERRABLE
+except ImportError:
+    from airflow.configuration import conf
+
+    DEFAULT_DEFERRABLE = conf.getboolean("operators", "default_deferrable", fallback=False)
+
```

Comment:
it's probably a good idea to add a `todo` for each of these saying to remove this compat logic when provider min version >= airlfow 2.7.0

Reply:
I am considering moving this logic to the base operator, as suggested by @uranusjr, in order to eliminate the need for this import. However, I am uncertain of best approach to address the possible compatibility issues outlined in https://github.com/apache/airflow/pull/31712#issuecomment-1580187407. Would bumping a major version for the providers be a suitable solution?

Reply:
Wouldn't that mean it this would only function on 2.7+ only? I think 2.4+ makes more sense, and a comment as a reminder for 2.7 makes sense too. That said, what do folks think about just having line 34 above in each provider file. Similar to the alt solution in the pr description. That'd be my vote I think.

Reply:
If we implement that in the base operator, this would be a 2.7+ only feature. I'm thinking of still handling the default value of `deferrable` in the base operator but keeping the rest of the logic (i.e., import `DEFAULT_DEFERRABLE` without removing the `deferrable` attribute in provider operators) for [2.4, 2.7) compatibility. Once we upgrade the providers' min version to >= 2.7, the providers can get rid of handling the default value of `deferrable`.

The following is what I'm going to do, and I'll keep them separate commits so that we can quickly revert some if I misunderstood anything.

1. Add a `TODO` comment to each of the providers
2. Implement the assignment logic in base operators